conf/bk_server.conf

```diff
@@ -137,6 +137,11 @@ extraServerComponents=
 # avoid the executor queue to grow indefinitely
 # maxPendingAddRequestsPerThread=10000
 
+# Use auto-throttling of the read-worker threads. This is done
+# to ensure the bookie is not using unlimited amount of memory
+# to respond to read-requests.
+# readWorkerThreadsThrottle=true
```

Comment:
```suggestion
# readWorkerThreadsThrottleEnabled=true
```

Reply:
Good point. Renamed to "ThrottlingEnabled"